src/test/java/com/example/bank/transfer/export/XlsxTransferHistoryGeneratorTest.java

```diff
@@ -1,83 +0,0 @@
-package com.example.bank.transfer.export;
-
-import com.example.bank.exception.ExceptionType;
-import com.example.bank.exception.RestException;
-import com.example.bank.transfer.TransferHistoryRepository;
-import com.example.bank.transfer.model.TransferHistory;
-import org.apache.poi.ss.usermodel.FillPatternType;
-import org.apache.poi.ss.usermodel.HorizontalAlignment;
-import org.apache.poi.ss.usermodel.IndexedColors;
-import org.apache.poi.ss.usermodel.Workbook;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.InjectMocks;
-import org.mockito.Mock;
-import org.mockito.junit.jupiter.MockitoExtension;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import static com.example.bank.transfer.export.WorkbookCreator.headerCellTitles;
-import static com.example.bank.transfer.export.WorkbookCreator.transferHistorySheetName;
-import static org.mockito.Mockito.when;
-
-@ExtendWith(MockitoExtension.class)
-class XlsxTransferHistoryGeneratorTest {
```

Comment:
- shouldThrowGenerateXlsxTransferHistory() → Błędne użycie when().thenThrow()
- shouldCheckSheetAndHeaderNamesAreValid() → Można dodać weryfikację, czy arkusz nie jest pusty
- shouldStyleHeader() → Można dodać asercję na liczbę nagłówków
-  shouldFillDataValues() → Można dodać weryfikację liczby wierszy
- Brak testu dla pustych danych
- Brak testu dla błędnego numeru konta
- @Mock dla XlsxTransferHistoryGenerator → Niekonieczne bo mozesz to normalnie stworzyc
-